src/helper/helper.js

```diff
@@ -0,0 +1,10 @@
+import jwt from "jsonwebtoken";
+
+const generateToken = (user) => {
+  const accessToken = jwt.sign(user, process.env.ACCESS_TOKEN_SECRET);
+  if (!accessToken) {
+    return { message: "Failed to generate token!" };
```

Comment:
I suggest you make your message understandable by the user. The user doesn't know what a token is!

Reply:
instead of helper folder rename it to utils